stock_packed_picking/wizards/stock_pack_products.py

```diff
@@ -0,0 +1,62 @@
+from odoo import fields, models
+
+
+class PackProducts(models.TransientModel):
+    _name = "stock.pack.products"
+    _description = "Pack Products"
+
+    package_name = fields.Char()
+    operation_type_id = fields.Many2one(
+        comodel_name="stock.picking.type",
+        string="Operation Type",
```

Comment:
[OCA Coding Guidlines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields)
If the technical name of the field (the variable name) is the same to the string of the label, don't put string parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalised

Reply:
Does this mean that in this case field label automatically will become "Operation Type Id"? If answer is yes, then I don't want this "Id" reminder at the end.

Reply:
Yes, it will cut off the trailing "_id" so `operation_type_id` will be converted to `Operation Type`